src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -49,6 +53,14 @@ public void EnsureCapacity(int capacity)
                 Grow(capacity - _pos);
         }
 
+        public void AppendDefault(int defaultCharCount)
+        {
+            int newCapacity = _pos + defaultCharCount;
+            if (newCapacity > _chars.Length)
+                Grow(defaultCharCount);
+            Length = newCapacity;
+        }
```

Comment:
This should not be a method on ValueStringBuilder.  You can do it in your code where you need it.